### DISCUSSION TITLE
Info events mentioned as errors in performance transaction

### DISCUSSION THREAD

**Discussion by delta-emil** [getsentry/support#29] [Support]
We added some level:info logs and enabled sending them to our Sentry server.

Now any performance transaction during which these info messages were logged , include a big red area which says "There is an error event associated with this transaction event." and below in the scope where it happens we get "An error event occurred in this transaction." and the event listed as "info" and a link to the event in the issues section.

The transaction's status is still "ok", but we're wondering if this is affecting the calculated Error Rate. Is the Error Rate based on the presence of logged events (which the performance transaction page seemingly calls "errors" regardless of their level) during that time? Or is it based purely on the transaction's status?

Or is this just a confusing visual thing on the performance transaction page? (i.e. it just shows all logs with the error visuals and text, but the transaction isn't counted as "having errors") If so, it would be nice if the Sentry server adjusted what is shown based on the max level of the events logged during the performance transaction.

We are using the C# official client library and a self-hosted Sentry server.

**Comment by souredoutlook** [comment]
Hi, @delta-emil if you can share a link to the transaction event (if you're using SaaS sentry) I can have a look. Events generated from logging typically have an event type of "default" and my intuition would suggest that this should not be considered an error. 

If the event type is the default, and we're flagging it as an error I'd like to investigate further to see if we can feed this back to the Performance team as a potential enhancement.

Let me know what you can share about the events and I'll do my best to put together some feedback for the team.
